.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,108 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["bug", "support_request"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+        #Start here
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO Version/Branch/tag
+      placeholder: 2022.3.0 / Master Branch / tag 2022.3.0
```

Comment:
some people even use older LTS releases like 2021.4

Reply:
I think it's just a placeholder, a text which disappears once you click the text field. I guess we can add "Example: ..." before that to further indicate that it's just a suggestion, and not a list of available options.